### PR TITLE
Remove profiles

### PR DESCRIPTION
**Pull request by Ozaq**

### CODE REVIEW

File: libsimulator/src/GCFMModel.cpp [hunk 1]
```diff
@@ -95,13 +92,19 @@ void GCFMModel::CheckDistanceConstraint(
     const NeighborhoodSearchType& neighborhoodSearch) const
 {
     const auto neighbors = neighborhoodSearch.GetNeighboringAgents(agent.pos, 2);
+    const auto& model = std::get<VelocityModelData>(agent.model);
     for(const auto& neighbor : neighbors) {
-        const auto spacing = AgentToAgentSpacing(agent, neighbor);
-        if(spacing <= 0) {
+        const auto& neighborModel = std::get<VelocityModelData>(agent.model);
+        const auto contanctDist = model.radius + neighborModel.radius;
+        const auto distance = (agent.pos - neighbor.pos).Norm();
```
Comment (schroedtert) [inline libsimulator/src/GCFMModel.cpp:99]: Is there a reason for not using `AgentToAgentSpacing` anymore?

File: libjupedsim/include/jupedsim/jupedsim.h [hunk 2]
```diff
@@ -482,21 +432,235 @@ JUPEDSIM_API void JPS_JourneyDescription_Free(JPS_JourneyDescription handle);
 /**
  * Opaque type of Generalized Centrifugal Force model state
  */
-typedef struct JPS_GeneralizedCentrifugalForceModelState_t const*
+typedef struct JPS_GeneralizedCentrifugalForceModelState_t*
     JPS_GeneralizedCentrifugalForceModelState;
 
+/**
+ * Read speed of this agent.
+ * @handle of the Agent to access.
```
Comment (schroedtert) [inline libjupedsim/include/jupedsim/jupedsim.h:440]: Isn't it `@param param_name` in doxygen?

Comment (Ozaq) [reply libjupedsim/include/jupedsim/jupedsim.h:440]: yes, on it

File: libsimulator/src/GCFMModel.cpp (hunk 1 above)
Comment (Ozaq) [reply libsimulator/src/GCFMModel.cpp:99]: good catch

**Review by schroedtert — APPROVED** [review]

**Pull request merged**